packages/react-components/react-portal/src/components/Portal/Portal.types.ts

```diff
@@ -16,9 +16,6 @@ export type PortalProps = {
 
 export type PortalState = Pick<PortalProps, 'children'> &
   Required<Pick<PortalProps, 'mountNode'>> & {
-    /** Indicates if a Portal should be rendered. */
```

Comment:
should this be deprecated instead?

Reply:
No, it's not a part of public API since our composition hooks are all unstable. If this were a prop then yes we should dreprecate it